tests/policy/wasm/demo/Makefile

```
@@ -0,0 +1,16 @@
+TARGET=wasm32-unknown-unknown
+NAME=policy_wasm_demo
+OUT=target/$(TARGET)/release/$(NAME).wasm
+
+.PHONY: all build clean
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)